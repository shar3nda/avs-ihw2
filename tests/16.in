C-pqL.#rcnkiNp+yI&
3X)FdH8urEkBS*c2B0}B#$.55lo$2Qi&H<l@9fnXk#vb80m~~4Ly hv$))6_1