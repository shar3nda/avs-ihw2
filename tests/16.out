Lowercase: 27, uppercase: 16